hazelcast/proxy/cp/atomic_long.py

```diff
@@ -145,13 +146,18 @@ def alter(self, function):
 
         Args:
             function (hazelcast.serialization.api.Portable or hazelcast.serialization.api.IdentifiedDataSerializable):
-                The function applied to the currently stored value.
+                The function that alters the currently stored value.
 
         Returns:
             hazelcast.future.Future[None]:
         """
+        check_not_none(function, "Function cannot be None")
```

Comment:
A potentially stupid question. Don't we need to validate number inputs in other methods, like `new_value` in `get_and_set`?

Reply:
In fact, it makes a lot of sense. We don't have type checks in most of the codebase, we can improve that incrementally but It makes sense to add them for the new features. I also added simple tests for invalid input cases